### PR TITLE
Add changelog section to contributing docs

### PR DESCRIPTION
**Pull request by frankier**

### CODE REVIEW

**Review by mortenpi — APPROVED** [review]
Good idea!

File: docs/src/contributing.md [hunk 1]
```diff
@@ -58,6 +58,12 @@ Follow the style of the surrounding text when making changes. When adding new fe
   * always use fenced code blocks instead of indented blocks;
   * follow the conventions outlined in the Julia documentation page on documentation.
 
+### Changelog
```
Comment (fredrikekre) [inline docs/src/contributing.md:61]: Should be a level 2 heading? Or did you intend to put it under "Style Guide"?
```suggestion
## Changelog
```

Comment (frankier) [reply docs/src/contributing.md:61]: I did intend that since it's a style guide for the change log. Do you definitely think it should be top level?

Comment (fredrikekre) [reply docs/src/contributing.md:61]: Ah okay, I thought this was about the changelog in general.

Comment (mortenpi) [reply docs/src/contributing.md:61]: I think it actually makes sense to have this on the top level --- I think it's generally a good idea to let people know that a CHANGELOG is a thing. So I'll just go ahead and make it `h2`.

**Pull request merged**